Controllers/UsersController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using awsomAPI.Models;
+
+namespace awsomAPI.Controllers
+{
+    [Route("/")]
+    [ApiController]
+    public class RolesController : ControllerBase
+    {
+        private readonly AwsomApiContext _context;

```

Comment:
_ is not neccesary for private members, although its very common.

Reply:
Using the convention.